### DISCUSSION TITLE
Bluetooth to TCP Stream

### DISCUSSION THREAD

**Discussion by rad-llc** [snapcast/snapcast#825] [Q&A]
Totally new to this.  I have setup a snapserver and a few clients streaming spotify (using raspotify).  Is it possible to simply setup another raspberry pi with bluetooth on it.  Use a phone (with Pandora for example), connect to bluetooth on pi and send the music stream from pi to the snapserver so that all the clients get the audio stream.  I want to do this on separate raspberry pi's, not the one with the snapserver running on it so I can place bluetooth pi audio receivers in various levels of the house and send the audio to all connected clients of only one snapserver instance.  Thanks for any help / instructions on how to do this.

**Comment by pimseb** [comment]
Did you find how to do this ? I'm trying to do the same.
I can't create a fifo file through network and I don't understand how TCP client / server are working
